src/openrct2-ui/windows/Options.cpp

```diff
@@ -1311,8 +1309,10 @@ static void window_options_dropdown(rct_window *w, rct_widgetindex widgetIndex,
             }
             break;
         case WIDX_SCALE_QUALITY_DROPDOWN:
-            if (dropdownIndex != gConfigGeneral.scale_quality) {
-                gConfigGeneral.scale_quality = (uint8)dropdownIndex;
+            // Note: offset by one to compensate for lack of NN option.
+            if ((dropdownIndex + 1) != gConfigGeneral.scale_quality)
+            {
+                gConfigGeneral.scale_quality = (uint8)dropdownIndex + 1;
```

Comment:
Don't like this way of doing +1 and -1. I'd just leave it out since we're breaking people's config anyway.

Reply:
Also, can you change the default setting to SNN?

Reply:
> Don't like this way of doing +1 and -1. I'd just leave it out since we're breaking people's config anyway.

It's to match the scaling options enum. 0 is NN -- internally, that still exists, of course.

> Also, can you change the default setting to SNN?

It was already the default in Config.{h,cpp}, last I checked?

Reply:
Do we need NN to stay in the scaling options enum? Is it still used for something?

Reply:
You're right in that we don't need the options enum to actively hold the value -- and it's removed from Options.cpp for that reason. Rather, it's the `SCALE_QUALITY` enum in Config.h that still holds a `SCALE_QUALITY_NN` value, as that's still used to enable the use of NN in e.g. `UiContext.cpp`...

Reply:
Hm. Well, let's go with how you did it, then.

Reply:
Alternatively, we could move the `SCALE_QUALITY_NN` into the last position in the enum so the values match, I guess? (Although, ideally, we'd have a proper matching system in place; relying on integer arithmetic for enums is a little hacky.)